hub/config/src/test/java/uk/gov/ida/hub/config/data/LevelsOfAssuranceConfigValidatorTest.java

```diff
@@ -82,6 +82,16 @@ public void shouldAllowLOA1TransactionConfiguration() {
         }
     }
 
+    @Test
+    public void shouldAllowLOA2AndLOA1TransactionConfiguration() {
+        Set<TransactionConfig> transactionsConfig = Set.of(loa2And1Transaction);
```

Comment:
I can't see where `loa2And1Transaction` is defined.

Reply:
Expanded the class - now I can see it. Has it always been there?

Reply:
Yes, because previously it was used for a test to ensure that this order of LOAs could not be used.